posts/2025-06-03-25.0.0.6-beta.adoc

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: "Backporting compatibility of Microprofile Health-4.0 and introducing file-based health checks in 25.0.0.6-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/navaneethsnair1
+author_github: https://github.com/navaneethsnair1
+seo-title: Backporting compatibility of Microprofile Health-4.0 and introducing file-based health checks in 25.0.0.6-beta - OpenLiberty.io
+seo-description: This beta release brings updates to backporting compatibility of mpHealth-4.0 and file-based health checks that were earlier released in 25.0.0.4-beta.
+blog_description: This beta release brings updates to backporting compatibility of mpHealth-4.0 and file-based health checks that were earlier released in 25.0.0.4-beta.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Backporting compatibility of Microprofile Health-4.0 and introducing file-based health checks in 25.0.0.6-beta
+Navaneeth S Nair <https://github.com/navaneethsnair1>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+This beta release brings updates to backporting compatibility of mpHealth-4.0 and file-based health checks that were earlier released in 25.0.0.4-beta.
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/31468
+// Contact/Reviewer: Channyboy
+// // // // // // // // 
+[#backport]
+== Backporting compatibility of mpHealth-4.0 for Java EE7 and EE8 environments and introducing file-based health checks
+
+
+In the previous link:https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2025/04/08/25.0.0.4-beta.html[25.0.0.4-beta] release, the Open Liberty server introduced the link:https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2025/04/08/25.0.0.4-beta.html#backport[file-based health check mechanism] for a MicroProfile Health 4.0 feature (`mpHealth-4.0`). In 25.0.0.4-beta, the configuration for this mechanism was introduced with the `server.xml` attribute `fileUpdateInterval` for the `<mpHealth>` element with an associated `MP_HEALTH_FILE_UPDATE_INTERVAL` environment variable. The configuration attribute has now been updated to `checkInterval` with an associated `MP_HEALTH_CHECK_INTERVAL` environment variable. Similar to before, this attribute needs to be configured to enable this alternative file-based health check mechanism. The configuration accepts a number followed by an optional `ms` or `s` to denote the time unit. If no time unit is specified, it defaults to seconds. Invalid values default the interval to 10 s. 
```

Comment:
It might be beneficial to add just a sentence or two to summarize that this feature is for doing health checks on applications deployed to the server. I had to go to that blog to confirm that thought. If we can save people a click and simply add a sentence to quickly explain the value of this, we can keep them from jumping between pages unnecessarily.